huaweicloud/services/acceptance/apigateway/resource_huaweicloud_api_gateway_environment_test.go

```diff
@@ -0,0 +1,97 @@
+package apigateway
+
+import (
+	"fmt"
+	"log"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/apigw/shared/v1/environments"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func getEnvironmentFunc(cfg *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	client, err := cfg.ApiGatewayV1Client(acceptance.HW_REGION_NAME)
+	envId := state.Primary.ID
+	log.Printf("[DEBUG] env id is : %s", envId)
+	if err != nil {
+		return nil, fmt.Errorf("error creating APIG client %s", err)
+	}
+
+	envs, err := environments.List(client, environments.ListOpts{})
+	if err != nil {
+		return nil, err
+	}
+	log.Printf("[DEBUG] envs is : %s", envs)
```

Comment:
```suggestion
	log.Printf("[DEBUG] List of shared APIG environments: %#v", envs)
```
HTTP request will print request, response and header details, usually we don't print the result.